charts/gardener/operator/templates/customresouredefintion.yaml

```
@@ -84,8 +84,169 @@ spec:
+                                      to reference a secret resource.
+                                    type: string
+                                  namespace:
+                                    description: namespace defines the space within
+                                      which the secret name must be unique.
+                                    type: string
+                                type: object
+                                x-kubernetes-map-type: atomic
+                            required:
+                            - bucketName
+                            - provider
+                            - secretRef
+                            type: object
+                          storage:
+                            description: Storage contains storage configuration.
+                            properties:
+                              capacity:
+                                anyOf:
+                                - type: integer
+                                - type: string
+                                default: 10Gi
+                                description: Capacity is the storage capacity for
+                                  the volumes.
+                                pattern: ^(\+|-)?(([0-9]+(\.[0-9]*)?)|(\.[0-9]+))(([KMGTPE]i)|[numkMGTPE]|([eE](\+|-)?(([0-9]+(\.[0-9]*)?)|(\.[0-9]+))))?$
+                                x-kubernetes-int-or-string: true
+                              className:
+                                description: ClassName is the name of a storage class.
+                                type: string
+                            type: object
+                        type: object
+                    type: object
+                  maintenance:
+                    description: Maintenance contains information about the time window
+                      for maintenance operations.
+                    properties:
+                      timeWindow:
+                        description: TimeWindow contains information about the time
+                          window for maintenance operations.
+                        properties:
+                          begin:
+                            description: Begin is the beginning of the time window
+                              in the format HHMMSS+ZONE, e.g. "220000+0100". If not
+                              present, a random value will be computed.
+                            pattern: '[0-2][0-3][0-5][0-9][0-5][0-9]\+[0-1][0-4]00'
+                            type: string
+                          end:
+                            description: End is the end of the time window in the
+                              format HHMMSS+ZONE, e.g. "220000+0100". If not present,
+                              the value will be computed based on the "Begin" value.
+                            pattern: '[0-2][0-3][0-5][0-9][0-5][0-9]\+[0-1][0-4]00'
```

Please see my comment before
```suggestion
                            pattern: '([0-1][0-9]|2[0-3])[0-5][0-9][0-5][0-9]\+[0-1][0-4]00'
```